taxcalc/tests/test_parameters.py

```diff
@@ -41,71 +42,96 @@ def test_create_parameters():
     assert p
 
 
+def test_params_json_content():
+    ppo = Parameters()
+    params = getattr(ppo, '_vals')
+    num_parameters = 0
+    num_missing_row_labels = 0
+    for name, data in params.items():
+        num_parameters += 1
+        start_year = data.get('start_year')
+        assert isinstance(start_year, int)
+        assert start_year == Parameters.JSON_START_YEAR
+        row_label = data.get('row_label')
+        if isinstance(row_label, list):
+            value = data.get('value')
+            expected = [str(start_year + i) for i in range(len(value))]
+            assert row_label == expected
+        else:
+            num_missing_row_labels += 1
+    if num_missing_row_labels != 0:
+        msg = 'num_parameters_in_total={}\n'
+        sys.stdout.write(msg.format(num_parameters))
+        msg = 'num_without_row_label_list={}\n'
+        sys.stdout.write(msg.format(num_missing_row_labels))
+        # assert num_missing_row_labels == 0
```

Comment:
I think it's fine to `assert` here or raise some kind of exception. I think it's also fine to `print` here, or if there is a desire to really direct this output somewhere, I would suggest `stderr`.